src/upload-tests.proj

```diff
@@ -102,8 +102,19 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyTestILCToTestHostDir" Condition="'$(BuildingUAPAOTVertical)' == 'true'">
+    <PropertyGroup>
+      <_TestILCFolderCopy>$(TestSharedFXDir)TestILC/</_TestILCFolderCopy>
```

Comment:
@MattGal would having the folders as a separate payload mean that helix builds and local builds will be different?